### ISSUE TITLE
Dialog Control: Narrator Reads all content present on dialog box

### ISSUE THREAD

**Issue by shrinidhizzz** [microsoft/fluentui#14467]
### Environment Information

#url: https://codepen.io/ShrinidhiMicrosoft/pen/bGpVBNm?editors=0010
Windows Version - Win10 
Chromium Edge Version: 85.0.552.1 (Official Build) dev (64-bit)

#### Repro Steps:

  * Go to https://codepen.io/ShrinidhiMicrosoft/pen/bGpVBNm?editors=0010 for a starter codepen
  * Turn on the screen reader tool
  * Listen what the reader reads when you enter to open the dialog open

#### Actual behavior:

As soon as "Remove Item" dialog control is activated the dialog gets opened and it starts announcing all the content present in the dialog such as "Remove Item dialog confirm button heading level 1 remove line item Do you want to remove the item from selection? "

#### Expected behavior:

The narrator should announce the only the "Remove
<img width="960" alt="dialogIssue" src="https://user-images.githubusercontent.com/30197247/89891248-5736d000-dbf2-11ea-9e33-cc393cae7eda.png">
 Item dialog opened" instead of announcing all the content.

### Documentation describing expected behavior

<img width="960" alt="dialogIssue" src="https://user-images.githubusercontent.com/30197247/89891365-92390380-dbf2-11ea-9f21-0bbcaa0a52ae.png">

**Comment by paulgildea** [comment]
@smhigley FYI

**Comment by khmakoto** [comment]
Hi @shrinidhizzz, if you look at the [ARIA spec](https://www.w3.org/WAI/GL/wiki/Using_ARIA_role%3Ddialog_to_implement_a_modal_dialog_box) and [its examples](https://www.w3.org/TR/wai-aria-practices/examples/dialog-modal/dialog.html) you can see that the `Dialog` behaves as expected. The screen reader reads the content until it arrives at the first actionable control, at which point it stops reading. Given that, I'll be closing this as `Resolution: Not an Issue`, but let us know if there is anything else we can do to help!
